sklearn/feature_extraction/text.py

```diff
@@ -771,16 +772,32 @@ def partial_fit(self, X, y=None):
         ----------
         X : ndarray of shape [n_samples, n_features]
             Training data.
+
+        y : None
+            This parameter is ignored.
```

Comment:
We tend to use the following in other PRs:

```suggestion
        y : Ignored
            Not used, present for API consistency by convention.
```

Reply:
Fixed